### PR TITLE
implement ibootstrap

### PR DESCRIPTION
**Pull request by blizzz**
requires
* [x] https://github.com/nextcloud/server/pull/22649
* [x] https://github.com/nextcloud/server/pull/22647

### CODE REVIEW

File: lib/Listener/LoadSettingsScriptsListener.php [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+			return;
+		}
+
+		$authorized = json_decode($this->config->getAppValue(Application::APP_ID, 'authorized', '["admin"]'));
+
+		$loadScript = true;
+		if(!empty($authorized)) {
+			$userGroups = $this->groupManager->getUserGroupIds($this->userSession->getUser());
+			if (!array_intersect($userGroups, $authorized)) {
+				$loadScript = false;
```
Comment (ChristophWurst) [inline lib/Listener/LoadSettingsScriptsListener.php:63]: just return early here and remove `loadScript` and the `if` below?

**Review by skjnldsv — APPROVED** [review]

File: js/admin_settings.js [hunk 2]
```diff
@@ -19,12 +19,12 @@
  */
 
 $(document).ready(function() {
```
Comment (skjnldsv) [inline js/admin_settings.js:21]: DOn't use ready anymore

**Review by skjnldsv — CHANGES_REQUESTED** [review]

File: js/admin_settings.js (hunk 2 above)
Comment (blizzz) [reply js/admin_settings.js:21]: okay, but -v? To get rid of jQuery totally, the post method also oughtta go.

Comment (skjnldsv) [reply js/admin_settings.js:21]: https://github.com/nextcloud/server/issues/20953#issue-617327468

Comment (blizzz) [reply js/admin_settings.js:21]: @skjnldsv i added a commit that gets rid of jQuery. Good?

Comment (blizzz) [reply js/admin_settings.js:21]: ah, but that was another file

**Review by ChristophWurst — APPROVED** [review]

File: js/admin_settings.js (hunk 2 above)
Comment (blizzz) [reply js/admin_settings.js:21]: @skjnldsv When I use ```js
if (document.readyState !== 'loading') {
		fnc()
	} else {
		document.addEventListener('DOMContentLoaded', fnc)
	}```
i run into a race condition that I am always loosing, OC.Settings will never be ready by that. Of course I can fumble around with setting timeouts again…

Comment (skjnldsv) [reply js/admin_settings.js]: `document.addEventListener('DOMContentLoaded', fnc)` is enough.
Are you sure you don't want to move to a compiled package?

Comment (blizzz) [reply js/admin_settings.js]: Figured it out shortly afterwards :)

Not for these few lines, really.

**Comment by nickvergessen** [comment]
Tests are failing?

**Comment by blizzz** [comment]
parallel-lint vanished from master it seems

**Pull request merged**